### ISSUE TITLE
Scripts in `./scripts` not being found?

### ISSUE THREAD

**Issue by ethanwhite** [weecology/retriever#623]
After running `retriever reset scripts` when I try to do an installation from inside the repository I get a dataset unavailable error. This shouldn't happen since the data is in the local `scripts` directory.

```
ethan@stice:~/RetrieverCode (master)$ retriever install csv MoM2003
The dataset MoM2003 isn't currently available in the Retriever
Run 'retriever ls to see a list of currently available datasets
```

There was a recent discussion about not making a change that would have removed this functionality. Maybe it slipped through in the refactoring around the datapackage changes?

**Comment by henrykironde** [comment]
I think that is because, we didn't include the search paths.  
I remember mention this in one of the PRs 
`ref: current master https://github.com/weecology/retriever/blob/master/__init__.py`
```
]
SCRIPT_WRITE_PATH = SCRIPT_SEARCH_PATHS[-1]
DATA_SEARCH_PATHS = [
    "./",
    "{dataset}",
    "raw_data/{dataset}",
    os.path.join(HOME_DIR, 'raw_data/{dataset}'),
]
```

**Comment by goelakash** [comment]
This should be closed now.

**Comment by ethanwhite** [comment]
Fixed by 88d6fcc5d8422c3b725dbe4b299679c35ceb9e63.
